tools/dhis2-audit-data-extractor/extract_audit.py

```diff
@@ -0,0 +1,74 @@
+import psycopg2
+import json
+import gzip
+
+DHIS2_CONF_FILE="/home/dhis/config/dhis.conf"
+CONN_CONFIG = {
+	"host": None,
+	"dbname": None,
+	"username": None,
+	"password": None
+}
+
+with open(DHIS2_CONF_FILE) as f:
+    lines = f.readlines()
+
+
+for line in lines:
+	split_line = line.split("= ")[1]
```

Comment:
`key=value` (without spaces between the equal sign) is valid syntax
```suggestion
	split_line = line.split("=")[1]
```

another option would be to use Python's config parser module (though since the `dhis.conf` does not use headers one of these workarounds will have to be used: https://stackoverflow.com/questions/2885190/using-configparser-to-read-a-file-without-section-name)

Reply:
@mediremi the line contains a space after the `=`, like `connection.url = <string>` so that space is needed to correctly use the `split()`.

I should use the `lstrip()` on that string, which I may be in the future, but your suggestion breaks the code so you should remove it.

Reply:
Ah ok, thanks for the clarification. I'll make sure to test few different use cases and perhaps implementing the suggested solution on the stackoverflow link.